app/views/main.html

```diff
@@ -117,54 +119,58 @@ <h2 class="nav-start">{{mainCtrl.message}}</h2>
-                    <div class="col-md-6">
-                        <div class="input-group col-md-6 col-md-offset-3">
+                    <div class="col-md-5">
+                        <div class="input-group col-md-6 col-md-offset-1">
                             <h2>Kup <span class="badge" id="newValueForBuy">{{mainCtrl.data.model}} {{mainCtrl.money}}</span>
                             </h2>
                             <div class="tour">
                                 <form name="form2" novalidate>
                                     <div>
                                         <input type="number" min="1" max="1000000" ng-model="mainCtrl.amountForBay"
                                                ng-change="mainCtrl.getCurrency(mainCtrl.data.model)" ng-model-options="sizese" name="sizese" integer/>
-                                        <button ng-click="mainCtrl.buy()">Zatwierdz</button>
+                                        <button ng-click="mainCtrl.buy()" class="btn btn-default main_accept" ng-disabled=" mainCtrl.amountForBay == undefined">Zatwierdz</button>
                                         <br/>
-                                        <p ng-show="form2.sizese.$error.min || form2.sizese.$error.max">
-                                            Podaj wartosc wieksza od 0 a mniejsza od 1 000 000!</p>
+                                        <p class="comunicatErr" ng-show="form2.sizese.$error.min">
+                                            Podaj wartosc wieksza od 0!</p>
+                                        <p class="comunicatErr" ng-show="form2.sizese.$error.max">
+                                            Podaj wartosc mniejsza od 1 000 000!</p>
                                     </div>
                                 </form>
                             </div>
                             <div class="input-group body-calc1">
-                                <p>Za <span id="pln11-count">{{mainCtrl.amountForBay}}</span> Trzeba zapłacić <span id="newWaluteForBuy">{{mainCtrl.buyInWindow}}</span>
+                                <p ng-show="mainCtrl.amountForBay > 0">Za <span id="pln11-count">{{mainCtrl.amountForBay || 0}}</span> Trzeba zapłacić <span id="newWaluteForBuy">{{mainCtrl.valueInViewBuy }}</span>
                                     PLN</p>
                             </div>
                         </div>
                     </div>
-                    <div class="col-md-6">
-                        <div class="input-group body-calc1 col-md-6">
+                    <div class="col-md-5">
+                        <div class="input-group body-calc1">
                             <h2>Sprzedaj <span class="badge">{{mainCtrl.data.model}} {{mainCtrl.moneyInSell}}</span>
                             </h2>
                             <div class="tour">
-                                <form name="form" novalidate>
+                                <form name="form1" novalidate>
```

Comment:
`form1`? is there `form2`? You should describe this form with name that tell us what kind this form is.